packages/types/src/types.ts

```diff
@@ -277,10 +276,10 @@ type TypesByFork = {
   [ForkName.gloas]: {
     BeaconBlockHeader: phase0.BeaconBlockHeader;
     SignedBeaconBlockHeader: phase0.SignedBeaconBlockHeader;
-    BeaconBlock: gloas.BeaconBlock;
-    BeaconBlockBody: electra.BeaconBlockBody;
-    BeaconState: gloas.BeaconState;
-    SignedBeaconBlock: gloas.SignedBeaconBlock;
+    BeaconBlock: electra.BeaconBlock; // TODO GLOAS: Use Gloas type
```

Comment:
This is temporarily set to electra because there is a lot of type issue that needs to be fixed with the latest definition of gloas.BeaconBlock and others